carsus/io/cmfgen/base.py

```diff
@@ -698,10 +819,40 @@ def _get_levels_lines(self, data):
             lns = lns.reset_index()
             lns_list.append(lns)
 
+            if 'phixs' in reader.keys():
+                if ion == (1,0):
+                    
+                    n_levels = 30
+                    l_points, l_start_u, l_del_u = 97, 0.0, 0.041392685
+                    n_points, n_start_u, n_del_u = 145, 0.0, 0.041392685
+
+                    hyd = reader['hyd'].apply(lambda x: 10 ** (8 + x))
+                    gbf = reader['gbf']
+
+                    hyd_phixs, hyd_phixs_energy_grid_ryd = {}, {}
+                    hyd_gaunt_factor, hyd_gaunt_energy_grid_ryd = {}, {}
+
+                    for n in range(1, n_levels+1):
+
+                        lambda_angstrom = lvl.loc[n-1, 'Lam(A)']
+                        e_threshold_ev = HC_IN_EV_ANGSTROM / lambda_angstrom
+
+                        hyd_gaunt_energy_grid_ryd[n] = [e_threshold_ev / RYD_TO_EV * 10 ** (n_start_u + n_del_u * i) for i in range(n_points)]
```

Comment:
with line 838: HC_IN_EV_ANGSTROM / RYD_TO_EV
we should remove unnecessary conversions